internal/utils/docker.go

```diff
@@ -281,7 +282,13 @@ func DockerStart(ctx context.Context, config container.Config, hostConfig contai
 		hostConfig.Binds = binds
 	}
 	// Create container from image
-	resp, err := Docker.ContainerCreate(ctx, &config, &hostConfig, nil, nil, containerName)
+	resp, err := Docker.ContainerCreate(ctx, &config, &hostConfig, &network.NetworkingConfig{
+		EndpointsConfig: map[string]*network.EndpointSettings{
+			NetId: {
+				Aliases: aliases,
```

Comment:
I wonder if we can simplify the implementation by automatically adding a network alias based on the container name. For eg.

```go
parts := strings.Split(strings.TrimSuffix(containerName, utils.Config.ProjectId), "_")
alias := strings.Join(parts[1:len(parts)-1], "_")
```

Would give us the following transformation
```
supabase_pooler_cli                => pooler
supabase_studio_cli                => studio
supabase_pg_meta_cli               => pg_meta
supabase_edge_runtime_cli          => edge_runtime
storage_imgproxy_cli               => imgproxy
supabase_storage_cli               => storage
supabase_rest_cli                  => rest
realtime-dev.supabase_realtime_cli => realtime
supabase_inbucket_cli              => inbucket
supabase_auth_cli                  => auth
supabase_kong_cli                  => kong
supabase_db_cli                    => db
```

Reply:
Yeah sure - originally I felt this approach could be a bit brittle, but it's definitely simpler and removes the need to change function signatures 👍

Only potential downside is we would be limited to a single alias for each container, in case we wanted to add more for some containers, like `api` or `api.supabase.internal` on Kong.

Reply:
> this approach could be a bit brittle

That's true. Perhaps it would be less brittle if we generate the container names from the list of aliases instead. That should give us more flexibility as we only need to guarantee that the aliases don't change.